Index: a.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>test
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/a.txt b/a.txt
--- a/a.txt	(revision c140bec7d0ac96f1dc2afa12965acac581f82e42)
+++ b/a.txt	(date 1623554355042)
@@ -1,1 +1,1 @@
-test
\ No newline at end of file
+test2343434
